programs/tensorflowDetection2D/main.cpp

```diff
@@ -79,6 +80,15 @@ int main(){
   std::cout<<"Initializing YARP network..."<<std::endl;
   yarp::os::Time::delay(1);
 
+  // Apertura puerto de recepcións
+  std::cout<<"Opening image input port with the name /tensorflowDetection2D/img:i."<<std::endl;
+  yarp::os::Time::delay(1);
+  //BufferedPort<yarp::sig::ImageOf<yarp::sig::PixelRgb> > inImg;
+
+  //inImg.open("/tensorflowDetection2D/img:i");
+  yarp::os::Time::delay(1);
```

Comment:
Can we remove all these delays?

Reply:
No problem, i´m using delays for testing, only to check everything loads correctly.

Reply:
Done at bd2c31935b3feac4c3a09605bb6eee7efe78fe1f.